.ci/docker/common/install_cpython.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
+        do_cpython_build $py_ver Python-$py_ver
```

Comment:
Umm, where does this line come from?

Reply:
Fixed, thank you for the thorough review!